### PR TITLE
Leroyg/issues/49 add links to social media pages

### PR DESCRIPTION
**Pull request by leroyg**
I added SM links to footer: Facebook, LinkedIn, Twitter, Medium. 
I used FontAwesome to explicitly call them individually and assigned them as const.
Prior state there were no SM links on the footer. 
Current state after changes is there are SM links both on desktop and responsive.

### CODE REVIEW

**Comment by ambergkim** [comment]
@leroyg 

Requested Changes:
- Can we make the icons bigger especially on mobile so it's easier for a big finger to click one?
- possibly more comment(s) in the diff.

For next PR's:
- shorten the branch names as much as possible (some DevOps tools have issues w/ long names)
- When making changes to front end stuff, it's best to add before and after screenshots
- Also, add a link to any issues and vice versa (link the PR in the issue).
- Try practicing squashing your commits into related ones (we need to be able to follow your solution for something but how you got there is not needed).

Anytime:
Also, if you find that our documentation and SOP's need to be updated, please update so it can be available for other volunteers. :)

File: src/components/footer/footer.jsx [hunk 1]
```diff
@@ -2,15 +2,37 @@ import React from 'react';
...
-    return <footer>
+    return (
+      <footer>
       <img className="footer-logo" src={logo} />
+
+      <div className="social-media">
+        <p>Follow us on Social Media</p>
+    
+          <div className="social-icons">
+            <a href="https://linkedin.com/company/teamnouri">{linkedinSocial}</a>
```
Comment (ambergkim) [inline src/components/footer/footer.jsx]: Read up on a tags and how to optimize them for accessibility and SEO.

Comment (kylemh) [reply src/components/footer/footer.jsx]: In terms of accessibility, you're rendering these links with SVGs as their "content". For somebody that can't see the symbols - they'll have no context of what they're being linked to. FontAwesome icons don't use a `title` element to provide context about their icon so it's up to you to provide text that a screen reader would parse.

I recommend creating a ScreenReaderOnly component that has invisible, but rendered text. Your seeing-eye users will see the icons, and your screen reader-using users will be given the proper context. A good example would be:

```
<a href="https://linkedin.com/company/teamnouri">
  <ScreenReaderOnly>teamnouri on LinkedIn</ScreenReaderOnly>
  {linkedinSocial}
</a>
```

This methodology will also help you earn points with SEO audits for having descriptive text in hyperlinks.

See: https://github.com/OperationCode/front-end/tree/master/components/ScreenReaderOnly

Comment (leroyg) [reply src/components/footer/footer.jsx]: @kylemh thank you for the specific guidance Kyle. I appreciate it. I am working to incorporate those changes. @ambergkim Thank you.

**Comment by RaevLogic** [comment]
Are we doing anything with this still or is it obsolete with the website overhaul?

**Pull request closed without merging**